src/org/zaproxy/zap/model/TechSet.java

```diff
@@ -49,10 +49,14 @@ public TechSet (Tech[] include, Tech[] exclude) {
 	}
 	
 	public TechSet(TechSet techSet){
+		this.merge(techSet);
+	}
+	
+	public void merge(TechSet techSet){

```

Comment:
The implementation doesn't look right, it's possible to have a technology included and excluded at the same time.